recipe/meta.yaml

```diff
@@ -49,7 +45,7 @@ test:
     - alembic.util
   requires:
     - pip
-    - python <3.10
+    - python
```

Comment:
If python is already a run requirement, why is it listed again as a test requirement? That seems redundant at best.

Reply:
The `python` test requirements was removed:
https://github.com/AnacondaRecipes/alembic-feedstock/blob/a5d22006d544374cb61048e2d3158ca5bf342421/recipe/meta.yaml#L46&L48